### PR TITLE
Page enumerator

### PR DESCRIPTION
**Pull request by PiezPiedPy**
Allows pages to be iterated through with a foreach loop

Also fixed a spelling mistake in OrderType

### CODE REVIEW

File: src/GdaxApi/GdaxApi/Pagination/Page.cs [hunk 1]
```diff
@@ -1,11 +1,57 @@
...
 
         public U Before { get; internal set; }
 
         public U After { get; internal set; }
+
+
+        //private enumerator class
+        private class PageEnumerator: IEnumerator
+        {
+            private T[] Items;
```
Comment (rotiv123) [inline src/GdaxApi/GdaxApi/Pagination/Page.cs]: Make readonly and use the proper "camelCase".

File: src/GdaxApi/GdaxApi/Pagination/Page.cs [hunk 2]
```diff
@@ -1,11 +1,57 @@
...
+    public class Page<T, U>: IEnumerable
     {
         public T[] Items { get; internal set; }
 
         public U Before { get; internal set; }
 
         public U After { get; internal set; }
+
+
+        //private enumerator class
```
Comment (rotiv123) [inline src/GdaxApi/GdaxApi/Pagination/Page.cs]: please remove unnecessary code comments

File: src/GdaxApi/GdaxApi/Pagination/Page.cs [hunk 3]
```diff
@@ -1,11 +1,57 @@
...
+
+            public object Current
+            {
+                get
+                {
+                    try
+                    {
+                        return Items[position];
+                    }
+
```
Comment (rotiv123) [inline src/GdaxApi/GdaxApi/Pagination/Page.cs]: please remove unnecessary empty lines

**Review by rotiv123 — CHANGES_REQUESTED** [review]

**Comment by PiezPiedPy** [comment]
@rotiv123 Applied the changes you requested

**Review by rotiv123 — APPROVED** [review]

**Pull request merged**